nix/ci-config.nix

```diff
@@ -70,13 +70,11 @@ in
     # NOTE(Emily, 22 Mar 2023): Here we provide sensible defaults for working with liqwid-nix. 
     # These can be overwritten by the user, though. In the future, we may want some out-of-the-
     # -box implementation for enabling this somehow.
-    flake = { ... }: {
+    herculesCI = {
       # Added in: 2.7.2.
```

Comment:
technically added in 2.9.1 but not super important

Reply:
Well, the functionality was added in 2.7.2 and preserved until now. If we update the version every time we modify it, it is basically useless information. IMO.

Reply:
I'd agree if we could guarantee that it works in exactly the same way but down the line if issues arise from this change, given that we version it according to my suggestion, we'll know on what version the breaking change happened.

Reply:
Well, I can add a comment for it.

Reply:
That sounds even better, something that explains that this was changed from 2.9.1 but that preserves the original version in which Hercules was added.